### ISSUE TITLE
Accessibility: Remove aria-label on plugin card

### ISSUE THREAD

**Issue by joedolson** [aspirepress/AspireExplorer#43]
The plugin card list item has `aria-label` with the plugin name. 

Remove.

As a note, in the majority of cases where `aria-label` is added to an element that is not focusable, it will be ignored. However, if it is *not* ignored, as in this case, then the `aria-label` would replace the entire inner content of the list item: e.g., the entire card would just be named as the plugin name, and all other details would be lost.

There's an ambiguity in that aria-labels are mostly only read when the element is focused, so elements that don't receive focus usually won't have it read. But it's only going to be truly ignored on elements where naming is prohibited, and `p`, `li`, etc., are not on the prohibited list. 

So this is *mostly* harmless, but it will never have a benefit and in an edge case could be extremely harmful.
